### PR TITLE
doc(pdm): document side-effects of JFrog OIDC auth and permissions

### PR DESCRIPTION
**Pull request by noirbizarre**
Add some documentation on JFrog Artifactory OIDC and attestations permissions side effects.

### CODE REVIEW

File: pdm/README.md [hunk 1]
```diff
@@ -15,3 +15,36 @@ Provides some reusable PDM actions.
...
+
+When enabled, the [Initialize `pdm` action (`init`)](init/) will authenticate against Artifactory using
+the [dedicated `jfrog-login` action](https://github.com/LedgerHQ/actions-security/tree/main/actions/jfrog-login).
+
+As a consequence, all jobs requiring access to JFrog Artifactory, even read-only, should have the permissions to issue OIDC token:
+
+As soon as you publish on Artifactory (a Python library or a Docker image), those actions will attest and sign them.
+As a consequence, you will also need the `attestations` permission.
+
+When defining custom permissions, you need to redefine the default one, at least the `contents: read` require to checkout a repository.
```
Comment (niooss-ledger) [inline pdm/README.md:39]: typo:
```diff
- at least the `contents: read` require to
+ at least the `contents: read` is required to
```

File: pdm/README.md [hunk 2]
```diff
@@ -15,3 +15,36 @@ Provides some reusable PDM actions.
...
+- a `JFROG_DOCKER_REPOSITORY` environment variable for Docker
+
+We rely on environment variable instead of explicit parameters because:
+- there is a hard limit of 10 parameters on reusable actions
+- `pyproject.toml` sources support environment variable inject for credentials and we use the same naming
+
+When enabled, the [Initialize `pdm` action (`init`)](init/) will authenticate against Artifactory using
+the [dedicated `jfrog-login` action](https://github.com/LedgerHQ/actions-security/tree/main/actions/jfrog-login).
+
+As a consequence, all jobs requiring access to JFrog Artifactory, even read-only, should have the permissions to issue OIDC token:
```
[the remark this answers is not in the document]
Comment (noirbizarre) [reply pdm/README.md]: Added the missing/moved statement.

File: pdm/README.md (hunk 1 above)
Comment (noirbizarre) [reply pdm/README.md]: Fixed 👍🏼

File: pdm/README.md [hunk 3]
```diff
@@ -15,3 +15,36 @@ Provides some reusable PDM actions.
...
+
+## JFrog Artifactory
+
+All those actions have opt-in support for both consuming from and publishing to out internal Artifactory repositories.
+
+The feature is enabled by the presence of:
+- a `JFROG_REPOSITORY` environment variable for Python dependencies
+- a `JFROG_DOCKER_REPOSITORY` environment variable for Docker
+
+We rely on environment variable instead of explicit parameters because:
```
Comment (niooss-ledger) [inline pdm/README.md:27]: typo: variable**s** (plural)

**Review by niooss-ledger — DISMISSED** [review]

File: pdm/README.md (hunk 3 above)
Comment (noirbizarre) [reply pdm/README.md]: Fixed 👍🏼

**Pull request merged**